app/src/main/java/org/greenstand/android/TreeTracker/userselect/UserSelectScreen.kt

```diff
@@ -71,22 +81,91 @@ fun UserSelectScreen(
             )
         }
     ) {
+        LazyVerticalGrid(
+            cells = GridCells.Fixed(2),
+            modifier = Modifier.padding(it),  // Padding for bottom bar.
+            contentPadding = PaddingValues(start = 8.dp, end = 8.dp, top = 10.dp)
+        ) {
+            items(state.planters) { user ->
+                UserButton(
+                    user = user,
+                    state = state,
+                ) { viewModel.selectUser(user) }
+            }
+        }
+    }
+}
 
+@Composable
+fun UserButton(
+    user: PlanterInfoEntity,
+    state: UserSelectState,
+    onClick: () -> Unit
+) {
+    val isSelected = state.selectedPlanter?.id == user.id
+
+    DepthButton(
+        onClick = onClick,
+        isSelected = isSelected,
+        modifier = Modifier
+            .size(width = 156.dp, height = 248.dp)
+            .padding(8.dp),
+        colors = DepthButtonColors(
+            color = AppColors.Gray,
+            shadowColor = if (isSelected) AppColors.GreenShadow else AppColors.GrayShadow,
+            disabledColor = AppColors.GrayShadow,
+            disabledShadowColor = AppColors.GrayShadow
+        )
+    ) {
         Column(
-            verticalArrangement = Arrangement.Center,
+            modifier = Modifier
+                .fillMaxHeight()
+                .fillMaxWidth()
+                .align(Alignment.TopCenter),
             horizontalAlignment = Alignment.CenterHorizontally
         ) {
-            Text("User Select")
-            state.planters.forEach { user ->
+            // User profile picture.
+            Box(  // TODO: Fetch user profile picture.
+                modifier = Modifier
+                    .padding(1.dp)
+                    .size(156.dp)
+                    .clip(RoundedCornerShape(5.dp))
+                    .background(AppColors.LightGray)
+            )
+
+            // User text data: Name, phone, and number of tokens.
+            Column(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .wrapContentHeight()
+                    .padding(horizontal = 12.dp, vertical = 2.dp)
+            ) {
                 Text(
-                    text = "${user.firstName} ${user.lastName}",
-                    modifier = Modifier
-                        .padding(16.dp)
-                        .clickable {
-                            viewModel.selectUser(user)
-                        }
-                        .background(color = if (state.selectedPlanter?.id == user.id) Color.Gray else Color.White)
+                    text = "Name\nPhone",  // TODO: Fetch user name and phone number.
```

Comment:
We have access to the name already, let's use that here to keep the functionality.